src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -110,106 +97,8 @@ public override BoundNode VisitITuplePattern(BoundITuplePattern node)
             return null;
         }
 
-        /// <summary>
-        /// Learn from any constant null patterns appearing in the pattern.
-        /// </summary>
-        /// <param name="inputType">Type type of the input expression (before nullable analysis).
-        /// Used to determine which types can contain null.</param>
-        private void LearnFromAnyNullPatterns(
-            int inputSlot,
-            TypeSymbol inputType,
-            BoundPattern pattern)
-        {
-            if (inputSlot <= 0)
-                return;
-
-            // https://github.com/dotnet/roslyn/issues/35041 We only need to do this when we're rewriting, so we
```

Comment:
:memo: the linked issue had already been closed. No more references to it in the code already.